### PR TITLE
Ready repo for crates.io release

### PR DESCRIPTION
**Pull request by tfanelli-rh**
All shared code between `sev` and `snp` are included in the library.

### CODE REVIEW

**Comment by connorkuehl** [comment]
@tfanelli-rh If you haven't already, please also go through the [crates.io packaging checklist](https://doc.rust-lang.org/cargo/reference/publishing.html#before-publishing-a-new-crate). Just the readiness checks, please don't push the crate to crates.io just yet.

**Comment by tfanelli-rh** [comment]
> @tfanelli-rh If you haven't already, please also go through the [crates.io packaging checklist](https://doc.rust-lang.org/cargo/reference/publishing.html#before-publishing-a-new-crate). Just the readiness checks, please don't push the crate to crates.io just yet.

Sure thing.

File: src/util.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+pub trait TypeLoad: Read {
+    fn load<T: Sized + Copy>(&mut self) -> Result<T> {
+        #[allow(clippy::uninit_assumed_init)]
+        let mut t = unsafe { MaybeUninit::uninit().assume_init() };
+        let p = &mut t as *mut T as *mut u8;
+        let s = unsafe { from_raw_parts_mut(p, size_of::<T>()) };
+        self.read_exact(s)?;
+        Ok(t)
+    }
```
Comment (haraldh) [inline src/util.rs:33]: again? Didn't you already correct this in https://github.com/enarx/snp/pull/7#discussion_r688317596 ??

**Review by haraldh — CHANGES_REQUESTED** [review]

File: src/util.rs (hunk 1 above)
Comment (tfanelli-rh) [reply src/util.rs]: I had that code from before the update and forgot to reflect the changes. Fixed.

**Comment by connorkuehl** [comment]
@tfanelli-rh could you configure the Cargo.toml to also include the badges that we see on other Enarx repos like [this](https://github.com/enarx/sev/)? Once they're added to the manifest, regenerating the README with cargo-readme should work.

Aside from that, this looks good to me. Even still, I'd like #3 to merge first so that this PR can rebase against a running CI. Since this PR will likely be the snapshot that is tagged and pushed to crates.io, I thought we'd get the CI merged first so we have a green CI for that tag.

**Review by haraldh — CHANGES_REQUESTED** [review]
yeah, please rebase to main/master for the CI to run

**Pull request merged**